CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+Version 2.0.9 (dev)
+-----------------------------
+
+- Fix ``UnboundLocalError: local variable 'key_cls' referenced before assignment`` for FK when using ``to`` as string pointing to model when models is a package
```

Comment:
Don't be shy and add your name (or github handle if you prefer) in brackets after this line. We like to show who contributed what.